### DISCUSSION TITLE
`Editable` textarea

### DISCUSSION THREAD

**Discussion by hazem3500** [chakra-ui/chakra-ui#3846] [Ideas]
currently, the Editable only works for `Input`,  are there any plans to have the same functionality for `Textarea`?

**Comment by adityaketkar** [comment]
> currently, the Editable only works for `Input`, are there any plans to have the same functionality for `Textarea`?

Hi @hazem3500 , you can use the following snippet to use `TextArea` instead of `Input` inside an Editable : 

`<EditableInput as="textarea" />`

**Comment by anubra266** [comment]
Take a look at this: https://gist.github.com/0-void/57ebc389b10f8ecab4f1cad0bf5d92b8

**Comment by heozeop** [comment]
If you want multi-line input for within editable context, 
take a look at this [pull request](https://github.com/chakra-ui/chakra-ui/pull/4443).

Before make any refactor commit, I want more opinion to consider.

Blow gist is temporal component for multi line input with editable context  
- [EditableWithTextarea](https://gist.github.com/heozeop/26785fbe95bde393be5c858645497867)
